Modules/hpe3par_snapshot.py

```diff
@@ -505,7 +637,18 @@ def main():
         },
         "rm_exp_time": {
             "type": "bool"
+        },
+        "schedule_name": {
+            "type": "str"
+        },
+        "task_freq": {
+            "type": "str",
+            "choices": ['yearl', 'monthly', 'weekly', 'daily', 'hourly']
```

Comment:
This does not take into account cron type option

And I guess it must be yearl'y'

Reply:
changed